frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.component.tsx

```diff
@@ -43,7 +43,7 @@ export const CoordsProperty = ({ value, label, onChange, onBlur, required, error
 	const prevValue = useRef(undefined);
 	const { getValues } = useFormContext();
 	const ticket = getValues() as ITicket;
-	const selectedTemplateId = ticket?.type ?? TicketsCardHooksSelectors.selectSelectedTemplateId();
+	const selectedTemplateId = TicketsCardHooksSelectors.selectSelectedTemplateId() ?? ticket?.type;
```

Comment:
why the change in order?

Reply:
This can cause a crash due to hook order if you refresh tabular view with a ticket (with a default pin) open
![image](https://github.com/user-attachments/assets/dc371666-c5cc-4b2d-a553-9f0ccfc167f0)
This change means the selector is always called so the order of hooks will always be the same